### PR TITLE
Product Entry-RoboParent

### PR DESCRIPTION
**Pull request by eringurley**

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -4,9 +4,94 @@
...
+                        <input type="range" name="chore-help" id="chore-help" min="0" max="5" required>
+                    </div>
+                    <div>
+                        <label class="most-helpful" for="most-helpful">
+                            Now, tell us what you think would be the most helpful thing your bot could help you with?
+                        </label>
+                        <textarea name="most-helpful" id="most-helpful">ex: I have eight loads of laundry a day and would like that to disappear...</textarea>
+                    </div>
+            </section>
+        <section>
```
Comment (Katedam) [inline form.html:86]: This section needs to indent one more time to be lined up with the section above

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <h1>Roboparent App</h1>
+    </header>
+    <main>
+    <h2>Have little ones? Need helping hands? We've got an app for that. </h2>
+    <p>Use our mobile app to tell us what you need.</p>
+    <p>We'll deploy our bots with your specific requests.</p>
+    <p>You can sleep in on Saturdays.</p>
+    <p> 
+        <a href ="form.html" id="customize-button">Customize Now!</a>
+    </p>
```
Comment (Katedam) [inline index.html:21]: lines 15-21 need to be indented 1 time

File: src/customize-product.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+function getProduct(formData) {
+    //convert data into variables
+    console.log(formData);
```
Comment (Katedam) [inline src/customize-product.js:3]: Remove this console.log :-)

File: form.html (hunk 1 above)
Comment (eringurley) [reply form.html:86]: completed!

File: index.html (hunk 2 above)
Comment (eringurley) [reply index.html:21]: done!

File: src/customize-product.js (hunk 3 above)
Comment (eringurley) [reply src/customize-product.js:3]: done!

**Pull request merged**